checking grammar on operations and ops accessing cluster page, updati… (#3075)
mesosphere/dcos-docs-site: pages/ksphere/konvoy/1.4/operations/index.md

enterprise: false
---

> After you deploy a Konvoy cluster and the platform services you want to use, you are ready to begin managing cluster operations and its application workloads to optimize your organization's productivity.

Reviewer: ```suggestion
After you deploy a Konvoy cluster and the platform services you want to use, you can begin managing cluster operations and its application workloads to optimize your organization's productivity.
```
